examples/03_renderable/main.rs

```diff
@@ -363,17 +363,14 @@ fn run() -> Result<(), Error> {
         .with::<ExampleSystem>(ExampleSystem, "example_system", &[])
         .with_bundle(TransformBundle::new().with_dep(&["example_system"]))?
         .with_bundle(RenderBundle::new())?
-        .with_local(create_render_system(
-            pipeline_builder,
-            Some(display_config),
-        )?)
+        .with_local(RenderSystem::build(pipeline_builder, Some(display_config))?)
```

Comment:
Doesn't really matter, but here "pipeline_builder" is used, while in the pong example it is "pipe".

Reply:
Fixing naming inconsistencies feels like something outside the scope of this PR.